Marlin/ubl_G29.cpp

```diff
@@ -691,13 +698,34 @@
 
     LEAVE:
 
+/*
+    #if ENABLED(ULTRA_LCD)
+      if (partial_mesh) {
+        lcd_reset_alert_level();
+        LCD_MESSAGEPGM("Partial Mesh!");
+        //lcd_setstatuspgm(PSTR("Partial Mesh!"));
+        lcd_quick_feedback();
+        partial_mesh = false;
+      }
+      else {
+        lcd_reset_alert_level();
+        LCD_MESSAGEPGM("G29 UBL done!");
+        //lcd_setstatuspgm(PSTR("G29 UBL done!"));
+        lcd_quick_feedback();
+      }
+    #endif
+*/
```

Comment:
Don't include commented-out code in a PR. Some debug output may be ok at times. Even when commenting-out things, preserve the indentation so naive text editors can fold blocks.